script.js

```diff
@@ -0,0 +1,90 @@
+/* Julie Rizzino
+script.js
+INFO 2134
+Thoendel
+3/20/20 */
+
```

Comment:
start this file with an window.addEventListener for load since our script tag is at the beginning of your html file.